index.html

```diff
@@ -5,12 +5,120 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="index.css">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
-     integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
     <title>Physics and Space science conference</title>
 </head>
 <body>
-    
+    <section>
+        <div class="link-bar">
+            <ul>
+                <li><a href=""><img src="./images/Vector4.png" class="alt-img"></a></li>
+                <li><a href=""><img src="./images/Vector2.png" class="alt-img"></a></li>
+                <li>Ikoote</li>
+                <li>MyPage</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+    </section>
+    <section class="menu-a">
+        <div class="nav-menu-a">
+            <div class="logs"><a href="./index.html"><img src="./images/img-logo.jpg"></a></div>
+            <div class="nav-a">
+                <a href="./about.html">About</a>
+                <a href="#program">Program</a>
+                <a href="#">Join</a>
+                <a href="#sponsers">Sponsors</a>
+                <a href="#speakers">Speakers</a>
+                <a href=""><button type="button" class="btn" id="tree">PSS campaign</button></a>
+            </div>
```

Comment:
- [OPTIONAL] As requested by the previous reviewer [here](https://github.com/ikoote1/Physics-and-Space-Science/pull/1#discussion_r1101404922) as optional, kindly adjust these anchor tags to be centered, for that you would need to do the following:

  - justify-content: center;
  - align-items: center;
  
And just in case it's the border for the last elements that is breaking the design, you can assign the same border to all the  other elements  too, and make it transparent  or white

- [OPTIONAL ] :thinking: In my opinion it would be better to write use ul and li tags for these nav items as they are constituting a list :recycle: